### PR TITLE
Implement libjn:values

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnValuesScalarEvaluator.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            result.set(abvs1);
+        } catch (IOException e) {
+            throw new SystemException(ErrorCode.SYSE0001, e);
+        } finally {
+            ppool.giveBack(tempTvp);
+            ppool.giveBack(sp);
+        }
+    }
+
+    private void addPairs(TaggedValuePointable tempTvp) throws IOException, SystemException {
```
Comment (prestoncarman) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnValuesScalarEvaluator.java]: Is this the same function from SimpleObjectUnionScalarEvaluator? If so, it seems like we should be sharing code here.

Comment (prestoncarman) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnValuesScalarEvaluator.java]: If not, maybe you could pick a better name.

Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/LibjnValuesScalarEvaluator.java]: I think that it's different as the SimpleObjectUnionScalarEvaluator also does duplicate elimination on the keys, but is might be good to call it "getValues(...)"

**Comment by prestoncarman** [comment]
Looks good, squash and prepare for commit.

**Pull request closed without merging**